packages/manager/modules/pci/src/projects/project/storages/containers/container/object/add/add.controller.js

```diff
@@ -30,6 +32,17 @@ export default class PciBlockStorageContainersContainerObjectAddController {
     this.files = [];
   }
 
+  showStorageClass() {
+    return (
+      this.container.regionDetails?.services?.length > 1 &&
+      this.container.regionDetails?.services?.some(
+        (item) =>
+          item.name === OBJECT_CONTAINER_OFFER_HIGH_PERFORMANCE &&
+          item.status === 'UP',
```

Comment:
to put in constant file to keep consistency as you're using constants for other values